jquery/index.d.ts

```diff
@@ -1220,7 +1220,7 @@ interface JQueryStatic {
      *
      * @param value The value to search for.
      * @param array An array through which to search.
-     * @param fromIndex he index of the array at which to begin the search. The default is 0, which will search the whole array.
+     * @param fromIndex the index of the array at which to begin the search. The default is 0, which will search the whole array.
```

Comment:
@johnnyreilly : I wanted to apply this recommendation but this pull request has been merged too quickly...
See #13730